7-layouts/styles.css

```diff
@@ -0,0 +1,109 @@
+    background-size: cover;
+    height: 305px;
+    width: 305px;
+}
+.art-4 {
+    background-image: url('https://t1.uc.ltmcdn.com/images/5/4/6/img_como_alimentar_a_un_perro_cachorro_4645_orig.jpg');
+    background-size: cover;
+    height: 305px;
+    width: 305px;
+}
+.art-5 {
+    background-image: url('https://www.mundoperro.net/wp-content/uploads/criar-a-un-cachorro-de-perro.jpg');
+    background-size: cover;
+    height: 305px;
+    width: 305px;
+}
+.art-6 {
+    background-image: url('https://www.okchicas.com/wp-content/uploads/2015/11/Adorables-cachorros-que-no-parecen-reales-23.jpg');
+    background-size: cover;
+    height: 305px;
+    width: 305px;
+}
+body {
+    margin: 0;
+}
+.head-1 {
+    color: white;
+    font-family: Arial, Helvetica, sans-serif;
+    grid-area: head-1;
+    padding-left: 30px;
+}
+.lorem {
+    color: white;
+    font-family: Arial, Helvetica, sans-serif;
+    grid-area: lorem;
+}
+.top-1 {
+    color: white;
+    background-color: black;
+    display: grid;
+    font-family: Arial, Helvetica, sans-serif;
+    grid-area: top-1;
+    padding: 10px;
+}
+@media screen and (max-width: 600px) and (min-width: 320px) {
+    header {
+        background-image: url('https://blog.dogvivant.com/wp-content/uploads/2015/05/Los-perros-son-carn%C3%ADvoros_Dog-Vivant_BLOG-Portada-700x467@2x.jpg');
+        background-size: cover;
+        column-gap: 30px;
+        display: grid;
+        grid-template-areas: 'top-1'
+                              'head-1'
+                              'lorem';
+    }
+    article{
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: space-around;
+        padding: 70px;
+    }
+  }
+
+@media screen and (max-width: 1023px) and (min-width: 601px) {
+    header {
+        background-image: url('https://blog.dogvivant.com/wp-content/uploads/2015/05/Los-perros-son-carn%C3%ADvoros_Dog-Vivant_BLOG-Portada-700x467@2x.jpg');
+        background-size: cover;
+        column-gap: 30px;
+        display: grid;
+        grid-template-areas: 'top-1 top-1'
+                             'head-1 head-1'
+                             'lorem lorem';
+    }
+    article{
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: space-around;
+        padding: 70px;
+}
+}
+@media screen and (min-width: 1024px) {
+    header {
+        background-image: url('https://blog.dogvivant.com/wp-content/uploads/2015/05/Los-perros-son-carn%C3%ADvoros_Dog-Vivant_BLOG-Portada-700x467@2x.jpg');
```

Comment:
You can put only once the `background-image` property outside the media queries.